### ISSUE TITLE
Move temporary Swarming task directory on Windows

### ISSUE THREAD

**Issue by erock2112** [luci/luci-py#268]
I'm running into path length limits on Windows when I attempt to add a CIPD package: https://chromium-swarm.appspot.com/user/task/2fbf2050bd91b510

In that example, the temporary Swarming task directory is c:\users\chrome~1\appdata\local\temp\1\isolated_runbx9xjw.
Could we shorten this to something like c:\b\t_bx9xjw ?

**Comment by nodirt** [comment]
cc @vadimsht

**Comment by maruel** [comment]
[fs.extend()](https://github.com/luci/luci-py/blob/master/client/utils/fs.py#L17) should be called in the corresponding cipd.py code, if relevant.

I'll make a separate CL to reduce `isolated_runXXX` to `irXXX`, which will save 10 characters (OMG!).

**Comment by maruel** [comment]
@erock2112 do you want to keep it open as to migrate the temporary directory to always be under the bot directory? I came to think it's a good idea anyhow.

**Comment by erock2112** [comment]
I haven't had a chance to test it out, but those 10 characters might not have done it all.  In general, I think it'd be great to use /b/tmp/abc123 instead of the current long path.

**Comment by maruel** [comment]
That was not enough, will shave off another 12 characters.

**Comment by vadimsht** [comment]
I also did this for the cipd side, but it hasn't been deployed yet: https://github.com/luci/luci-go/commit/e1a0627cdf3f8ea84b56f8b104524246ec6214e2
